provisioning/local/aws.tf

```diff
@@ -0,0 +1,61 @@
+locals {
+  aws_region = "eu-central-1"
+}
+
+provider "aws" {
+  profile             = "Keboola-Dev-Platform-Services-AWSAdministratorAccess"
```

Comment:
Myslím, že zatím jsme všude nastavili ten profil přes env a měli tu jen `allowed_account_ids` 🤔

Reply:
Nope
* https://github.com/keboola/sandboxes-service/blob/fe0b6fe9b107c3c8a4ae3fb2edc131d5d11b29bc/provisioning/local/aws.tf#L2
* https://github.com/keboola/ai-service/blob/02a7acbd7cf709d9d0bdd7dddaaf3cc6eddb90b8/provisioning/local/aws.tf#L2
* https://github.com/keboola/oauth-service/blob/b7902a0b25a5954ecfb7bc10579068c9106d78ab/provisioning/local/aws.tf#L6

Samozrejme je k diskuzi, nakolik je to uztecny, protoze pokud nebudes mit presne tenhle profil nastaveny v ENV, failne ti TF na tom, ze se nedostane na state backend. Ale to bych sem ted asi netahal.

Reply:
No právě proto že stejně ten profil potřebuješ mít nastavený pro remote state backend, tak je to tady imho zbytečný. Já jsem to nikde takhle nedělal :)

Reply:
Myslim si, ze nevadi, ze si to tam nekde nedal. Mam ted spoustu feedbacku z prubehu GCP, takze po dokonceni grouponu bych rad obecne refreshnul best-practices.